README.md

```diff
@@ -1,339 +1,108 @@
-
-### Environment based SSO
-
-A simple approach to test environment based SSO with the user_saml app is to use apache basic auth with the following configuration:
-
-```
-
-<Location /login>
-	AuthType Basic
-	AuthName "SAML"
-	AuthUserFile /var/www/html/.htpasswd
-	Require valid-user
-</Location>
-<Location /index.php/login>
-	AuthType Basic
-	AuthName "SAML"
-	AuthUserFile /var/www/html/.htpasswd
-	Require valid-user
-</Location>
-<Location /index.php/apps/user_saml/saml/login>
-	AuthType Basic
-	AuthName "SAML"
-	AuthUserFile /var/www/html/.htpasswd
-	Require valid-user
-</Location>
-<Location /apps/user_saml/saml/login>
-	AuthType Basic
-	AuthName "SAML"
-	AuthUserFile /var/www/html/.htpasswd
-	Require valid-user
-</Location>
-```
-
-## Fulltextsearch
-
-```
-docker-compose up -d elasticsearch elasticsearch-ui
-```
-
-- Address for configuring in Nextcloud: `http://elastic:elastic@elasticsearch:9200`
-- Adress to access elastic search from outside: `http://elastic:elastic@elasticsearch.local`
-- Address for accessing the ui: http://elasticsearch-ui.local/
-
-`sudo sysctl -w vm.max_map_count=262144`
-
-
-
-## Object storage
-
-Primary object storage can be enabled by setting the `PRIMARY=minio` environment variable either in your .env file or in docker-compose.yml for individual containers.
-
-```bash
-docker-composer up proxy nextcloud minio
-```
-
-## Development
-
-### OCC
-
-Run inside of the Nextcloud container:
-```
-set XDEBUG_CONFIG=idekey=PHPSTORM
-sudo -E -u www-data php -dxdebug.remote_host=192.168.21.1 occ
-```
-
-### Useful commands
-
-- Restart apache to reload php configuration without a full container restart: `docker-compose kill -s USR1 nextcloud`
-- Access to mysql console: `mysql -h $(docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' nextcloud_database-mysql_1) -P 3306 -u nextcloud -pnextcloud`
-- Run an LDAP search: `ldapsearch -x -H ldap://$(docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' nextcloud_ldap_1) -D "cn=admin,dc=planetexpress,dc=com" -w admin -b "dc=planetexpress,dc=com" -s subtree <filter> <attrs>`
-
-## Keycloak
-
-- Keycloak is using ldap as a user backend (make sure the ldap container is also running)
-- `occ user_oidc:provider Keycloak -c nextcloud -s 09e3c268-d8bc-42f1-b7c6-74d307ef5fde -d https://keycloak.local.dev.bitgrid.net/auth/realms/Example/.well-known/openid-configuration`
-- https://keycloak.local.dev.bitgrid.net/auth/realms/Example/.well-known/openid-configuration
-- nextcloud
-- 09e3c268-d8bc-42f1-b7c6-74d307ef5fde
-
-## Global scale
-
-```
-docker-compose up -d proxy portal gs1 gs2 lookup database-mysql
-```
-
-Users are named the same as the instance name, e.g. gs1, gs2
-
-## Imaginary
-
-Enable the imaginary server for generating previews
-
-```bash
-docker-composer up proxy nextcloud previews_hpb
-./scripts/enable-preview-imaginary.sh
-```
-
+These are features where you can use :
 
+- ☁ Nextcloud
```

Comment:
Maybe we can add this list and the sections from the readme as a table of contents to the start of the file?

Reply:
Yes, I agree with you ! And we can add the rest of documentation (`docs/`)

What do you think ?

Reply:
Okay, I created a table of contents for the READE.md for the moment : https://github.com/juliushaertl/nextcloud-docker-dev/pull/87/commits/9feb2adf57bfdcc9fe0d36ff472c012b5545db13